ui/lib/store.js

```diff
@@ -11,8 +11,8 @@ const initialState = fromJS({
   user: {},
```

Comment:
let's keep the changes in the meshmap branch specific to meshmap. check-in any other changes to the master branch.